.rubocop.yml

```diff
@@ -105,6 +105,9 @@ Lint/ShadowingOuterLocalVariable:
 Lint/SuppressedException:
   Enabled: false
 
+Lint/RedundantSafeNavigation:
+  Enabled: false
```

Comment:
Also make sure to update the hash in the ruby_rubocop.rb file, otherwise that package will no longer install.

Reply:
Done